src/Math/babylon.math.ts

```diff
@@ -387,7 +383,7 @@ module BABYLON {
          * @returns a new Color3 object
          */
         public static FromInts(r: number, g: number, b: number): Color3 {
-            return new Color3(r / 255.0, g / 255.0, b / 255.0);
+            return new Color3(r / 255, g / 255, b / 255);

```

Comment:
Removed all the  useless `.0`, it only adds noise.
Also it suggest that there might be a difference between floating point value and integer in javascript, which is wrong, everything is number